services/validation/internal/service/validation_service_test.go

```diff
@@ -25,4 +24,18 @@ func TestGetLinkedSchemas(t *testing.T) {
 
 	_, ok = getLinkedSchemas(map[string]interface{}{"linked_schemas": []string{}})
 	assert.Equal(t, false, ok)
+
+	_, ok = getLinkedSchemas(map[string]interface{}{"linked_schemas": []string{"https://ic3.dev/test2.json"}})
+	assert.Equal(t, false, ok)
+
+	_, ok = getLinkedSchemas(map[string]interface{}{"linked_schemas": []interface{}{"https://ic3.dev/test2.json"}})
+	assert.Equal(t, true, ok)
+
+	linkedSchemas, ok := getLinkedSchemas(map[string]interface{}{"linked_schemas": []interface{}{"https://ic3.dev/test2.json"}})
+	assert.Equal(t, "https://ic3.dev/test2.json", linkedSchemas[0])
+}
+
+func TestGetSchemaURL(t *testing.T) {
+	url := getSchemaURL("test1.json")
+	assert.Equal(t, "/schemas/test1.json.json", url)
```

Comment:
`.json.json`?

Reply:
Sorry. Should not include `.json` in the input.

Reply:
We don't have the test environment variables and I have created an issue for it. #53